src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -353,11 +353,14 @@ public async Task UpdatePeriodicBackup()
                 beforeSetupConfiguration: (string databaseName, ref BlittableJsonReaderObject readerObject, JsonOperationContext context) =>
                 {
                     var configuration = JsonDeserializationCluster.PeriodicBackupConfiguration(readerObject);
+                    var feature = HttpContext.Features.Get<IHttpAuthenticationFeature>() as RavenServer.AuthenticateConnection;
 
                     ServerStore.LicenseManager.AssertCanAddPeriodicBackup(configuration);
                     BackupConfigurationHelper.UpdateLocalPathIfNeeded(configuration, ServerStore);
                     BackupConfigurationHelper.AssertBackupConfiguration(configuration);
                     BackupConfigurationHelper.AssertDestinationAndRegionAreAllowed(configuration, ServerStore);
+                    if (feature != null)
+                        SecurityClearanceValidator.AssertSecurityClearance(configuration, feature.Status);
```

Comment:
remove the `if`, you can do instead:
```
 SecurityClearanceValidator.AssertSecurityClearance(configuration, feature?.Status);
```
and handle that internally

Reply:
Done